src/ext/java/org/opentripplanner/ext/flex/FlexConfig.java

```diff
@@ -0,0 +1,12 @@
+package org.opentripplanner.ext.flex;
+
+import org.opentripplanner.standalone.config.NodeAdapter;
+
+public class FlexConfig {
+  public static final int DEFAULT_MAX_TRANSFER_METERS = 1000;
+  public final int maxTransferMeters;
+
+  public FlexConfig(NodeAdapter json) {
+    maxTransferMeters = json.asInt("maxTransferMeters", DEFAULT_MAX_TRANSFER_METERS);
+  }
```

Comment:
The `src/ext/java/org/opentripplanner/ext/flex` package should not have a dependency on the `NodeAdaptor`.  Make a "normal" constructor here and a mapper in the `org.opentripplanner.standalone.config.sandbox` package. 

The reason is that we try to prevent cyclic dependencies, and improve modularity. E.g. if you need to create a `FlexConfig` object in a test, then this would make a bit awkward.